pkg/kubectl/cmd/apply.go

```diff
@@ -753,7 +753,7 @@ func (p *patcher) patch(current runtime.Object, modified []byte, source, namespa
 		}
 		patchBytes, patchObject, err = p.patchSimple(current, modified, source, namespace, name, errOut)
 	}
-	if err != nil && errors.IsConflict(err) && p.force {
+	if err != nil && (errors.IsConflict(err) || errors.IsInvalid(err)) && p.force {
```

Comment:
super nit: @dixudx how do you think about pruing `err != nil` here? it might be unneccesary to do the nil value check here because neither `IsConflict ` or `IsInvalid ` would return true on a nil error AFAICT. Not really sure if the additional judge here is for a better human-read :)

Reply:
@yue9944882 Although both `IsConflict` and `IsInvalid` return `False` for `nil.(Type)`, I'd prefer to keep it for several reasons,

1. Like you said, for better human readable and code convention.
2. `ReasonForError(nil)` will return `StatusReasonUnknown`, which is not right and confusing. `Nil` is different from status code 500.

Reply:
reasonable :)